README.md

```diff
@@ -122,6 +122,12 @@ Settings:
 
 6. Visit http://localhost:8000/sdl.php
 
+### Usage with docker
+1. Build locally: `docker build -t goSDL .`
+
+2. Run it: `docker run -ti --rm --env-file <your dotenv> -p 8080:8080 goSDL`
```

Comment:
Same like above, getting an error of lowercase `goSDL`. Should we change this to?
`docker run -ti --rm --env-file <your dotenv> -p 8080:8080 gosdl`